css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: 0.5s;
+}
+
+.card-wrapper {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+  transform: 0.5s;
+}
+
+.card > img {
+  width: 100%;
+}
+
+.custom-button:hover {
+  background-color: red;
+}
+
+.card:hover {
+  transform: scale(1.1);
+}
```

Comment:
The `.card` scale changes on hover should be a smooth transition:

---

![image](https://user-images.githubusercontent.com/16143771/146244603-e2e89db1-35dc-45db-a2c1-9ca03ead3747.png)